app/pages/_app.tsx

```diff
@@ -26,9 +52,17 @@ export default function MyApp({ Component, pageProps }) {
 
         <noscript>You need to enable JavaScript to run this app.</noscript>
       </Head>
+
       <IntlWrapper messages={en}>
         <Header
           isMain={isMain}
+          displaySidebar={(state) => setShowSidebar(state)}
+          showSidebar={showSidebar}
+        />
+        <Sidebar
+          config={config}
+          visible={showSidebar}
```

Comment:
Unclear how to use `visible` and `isDocs` props. Unclear how to add styles based on these props.